starter-code/js/index.js

```diff
@@ -50,3 +52,35 @@ window.onload = function(){
     deleteButtons[i].onclick = deleteItem;
   }
 };
+*/
+
+function main () {
+
+var costArray = document.querySelectorAll(".cost");
+var quantityArray = document.querySelectorAll(".quantity");
+var totalPrice = 0;
+
+
+var calculateButton = document.getElementById("calculate-btn");
+var totalAmmount = document.querySelector(".total");
+
+  var calculatePrice = function() {
+      for (var i = 0; i < quantityArray.length; i++) {
+        totalPrice+= costArray[i] * quantityArray[i];
+      }
+      // calculateButton.removeEventListener
+      console.log("monkeyBusiness");
+     totalAmmount.innerHTML = totalPrice;
+  };
+  
+  calculateButton.addEventListener("click", calculatePrice )
+                 
+        
+  
+
+                 
+                 
+                 
+}
+
+main()
```

Comment:
You don't need a main() function to be executed. The functions you declare will be executed once you trigger an event.